src/simplify.rs

```diff
@@ -0,0 +1,64 @@
+use core::ops::Div;
+
+use crate::{fraction::Fraction, gcd::Gcd, Quantity, Unit};
+
+/// Simplify fraction.
+///
+/// ## Examples
+/// ```
+/// use typed_phy::{fraction::Fraction, simplify::Simplify};
+///
+/// use typenum::{assert_type_eq, U12, U3, U32, U8};
+///
+/// type Simplified = <Fraction<U32, U12> as Simplify>::Output;
+/// assert_type_eq!(Simplified, Fraction::<U8, U3>);
+/// ```
+pub trait Simplify {
```

Comment:
Btw I didn't check GCD because it's big :neutral_face:

Reply:
Ok. GCD is a very hard feature :sweat_smile: